README.md

```diff
@@ -89,6 +88,7 @@ We need to create and use an automated Google Cloud account to write to Drive an
 - When you download the credentials JSON file, rename it "creds.json" and put it in the same directory as these scripts.
 - In the Google Cloud console website, use the left navigation pane to go to "APIs and Services"
 - Next, click "Enable APIs and Services". This will take you to the API library. You need to search for and enable the "Google Drive API" and "Google Sheets API". For each, search for it, click the term, and enable it.
+- Next, For OAuth create a key on https://support.google.com/cloud/answer/6158849?hl=en and store as client_secret.json
```

Comment:
Please mention that here we're discussing OAuth in the context of logging in to GMail. It's best to be clear, since we can't depend on the technical aptitude of users

Reply:
Updated information